styleguide/sample-pages/advice-collection/_advice_collection.html.haml

```diff
@@ -4,7 +4,7 @@
   .cads-grid-row
     .cads-grid-col-12
       = render partial: "@citizensadvice/design-system/haml/breadcrumb", locals: {breadcrumbs: {"breadcrumb_links" => [{"url" => "/link", "title" => "Tables"}]}}
-%main.cads-main-content
+%main.cads-main-content.cads-prose
```

Comment:
Could we update this to reflect the narrow scoping from https://github.com/citizensadvice/public-website/pull/458?